src/CMakeLists.txt

```diff
@@ -22,23 +22,29 @@ set_target_properties           (leptonica PROPERTIES OUTPUT_NAME       leptonic
 set_target_properties           (leptonica PROPERTIES DEBUG_OUTPUT_NAME leptonica-${VERSION_PLAIN}d)
 endif()
 
-if (NOT STATIC)
+if (BUILD_SHARED_LIBS)
     target_compile_definitions  (leptonica PRIVATE -DLIBLEPT_EXPORTS)
 endif()
 
+target_include_directories(
+    leptonica
+    INTERFACE
+    "$<INSTALL_INTERFACE:include/leptonica>"
```

Comment:
Are you sure about this one? Usually it's just `include`. With this configuration includes like `#include <mystuff/a.hpp>` will try `include/leptonica/mystuff/a.hpp` too.